stubs/chevron/chevron/main.pyi

```diff
@@ -0,0 +1,9 @@
+from _typeshed import StrOrBytesPath
+
+from metadata import version as version
+from renderer import render as render
```

Comment:
This kind of imports mean that you're supposed to access `chevron.main.version` and `chevron.main.render`. This doesn't seem intentional.

You can just delete these imports. The same applies to `renderer.py`.